conda_libmamba_solver/mamba_utils.py

```diff
@@ -84,8 +86,31 @@ def set_channel_priorities(index: Dict[str, "_ChannelRepoInfo"], has_priority: b
     return index
 
 
+_once = False
+
+
 def init_api_context() -> api.Context:
-    api_ctx = api.Context()
+    global _once
+    if hasattr(api.Context, "use_default_signal_handler"):
+        api.Context.use_default_signal_handler(False)
```

Comment:
I would support calling this and forgetting about the workaround.

Reply:
Agreed. I'll push the change. Thanks!